### PR TITLE
backupdb: Force db management enabled

### LINKED ISSUE #40: click-odoo-backupdb fails if list_db is False in config file

### PR DESCRIPTION
**Pull request by lmignon**
This is required to avoid error if list_db is False in config file.
fixes #40

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -6,4 +6,4 @@ force_grid_wrap=0
 combine_as_imports=True
 use_parentheses=True
 line_length=88
-known_third_party = click,click_odoo,mock,psycopg2,pytest,setuptools
+known_third_party = click,click_odoo,decorator,mock,psycopg2,pytest,setuptools
```
Comment (sbidoul) [inline .isort.cfg]: is it a third party?

File: click_odoo_contrib/backupdb.py [hunk 2]
```diff
@@ -11,11 +11,25 @@
...
+def db_management_enabled(method):
+    def force_db_management_enabled(method, self, *args, **kwargs):
+        list_db = odoo.tools.config["list_db"]
+        odoo.tools.config["list_db"] = True
+        try:
+            return method(self, *args, **kwargs)
+        finally:
+            odoo.tools.config["list_db"] = list_db
+
+    return decorator(force_db_management_enabled, method)
```
Comment (sbidoul) [inline click_odoo_contrib/backupdb.py]: a context manager would be a tad more flexible (we could use it in [other contexts](https://github.com/acsone/click-odoo-contrib/commit/826f663d100156b883b19d33ca11adfed4e7eefd)).

Comment (lmignon) [reply click_odoo_contrib/backupdb.py]: OK code adapted

**Comment by sbidoul** [comment]
@lmignon could you rebase this one?

**Comment by lmignon** [comment]
@sbidoul Rebase done...

**Comment by sbidoul** [comment]
Thanks!

**Pull request merged**